packages/plugins/aws-storage/package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@verdaccio/commons-api": "workspace:11.0.0-alpha.3",
     "@verdaccio/streams": "workspace:11.0.0-alpha.3",
-    "aws-sdk": "2.980.0"
+    "aws-sdk": "2.981.0"
```

Comment:
Just fyi, this dep does not cause much changes, https://github.com/verdaccio/verdaccio/pull/2218 should move to the v3 already which has better TS support and much better API.